src/device.cpp

```diff
@@ -166,23 +166,31 @@ matcher_factory::create_timestamp_composite_matcher( std::vector< std::shared_pt
     return std::make_shared<timestamp_composite_matcher>(matchers);
 }
 
-device::device(std::shared_ptr<context> ctx,
+device::device( std::shared_ptr<context> ctx,
```

Comment:
Can we avoid unrelated format changes for this PR?
Lets focus on the main changes

Reply:
Done.

Although I do prefer to fix bad formatting where I make changes anyway. Could be more easily compared in the VSCode view :-)